### DISCUSSION TITLE
hackathon para hace un proyecto super chinghon

### DISCUSSION THREAD

**Discussion by cmjosueh** [Launch-X-Latam/MisionFrontEnd#498] [Team Posts]
Tengo mas conocimientos en el backend, en bases de datos tengo experiencia en MySQL y PostgreSQL, el lenguaje de programación: Python y un año de experiencia con la plataforma de Azure, estuve en el curso AZ-500 y obtube los 80 puntos para la rifa de certificación. En los comentarios pon el nombre de tu usuario de git para que hagamos equipo

**Comment by VAlonzoGO** [comment]
pásame tu usuario ya estoy creando el equipo en teams

**Comment by cmjosueh** [comment]
cmjosueh

**Comment by cmjosueh** [comment]
aunque sugiero que todo lo hagamos aqui para practicar los comandos de git

**Comment by VAlonzoGO** [comment]
pásame tu correo de innovaccion

**Comment by DeusGarrix4k** [comment]
@VAlonzoGO me puedes agregar a tu equipo, no me entero de nada soy nuevo completamente en esto mi correo es: lx15821@innovaccion.mx gracias espero podamos trabajar en equipo

**Comment by geraodst117** [comment]
Geraodst117

**Comment by cmjosueh** [comment]
lx01843@innovaccion.mx

**Comment by DanniYos** [comment]
DanniYos mi usuario de github

**Comment by DanielBustillosVillar** [comment]
a mi tambien si pueden me agregan por fa mi correo es  lx05538
***@***.******@***.*** ***@***.***>

**Comment by snowingt** [comment]
hey me gustaria estar en el grupo my user es snowingt
y mi correo de teams lx08891@innovaccion.mx

**Comment by Eduar2TC** [comment]
Eduar2tc

**Comment by JoselinMiel** [comment]
Hola compañero quisiera estar en tu grupo no tendré todo tu conocimiento, pero lo que no se lo consulto hasta realizarlo y sacarlo adelante. Muchas gracias

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: cmjosueh ***@***.***>
Sent: Tuesday, November 8, 2022 12:37:20 PM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] hackathon para hace un proyecto super chinghon (#27)


Tengo mas conocimientos en el backend, en bases de datos tengo experiencia en MySQL y PostgreSQL, el lenguaje de programación: Python y un año de experiencia con la plataforma de Azure, estuve en el curso AZ-500 y obtube los 80 puntos para la rifa de certificación. En los comentarios pon el nombre de tu usuario de git para que hagamos equipo

**Comment by EliasQuino** [comment]
Hola mi name en github es: EliasQuino ,correo de innovacion: lx08662@innovaccion.mx   que la fuerza te acompañe, vamos por más

**Comment by giankpetrov** [comment]
lx09705@innovaccion.mx

**Comment by laputorres** [comment]
Hola me gustaria estar en el grupo my usuario es laputorres
y mi correo es lx14852@innovaccion.mx

**Comment by stvnmc** [comment]
stvnmc
github
.
teams
***@***.*** ***@***.***>

**Comment by StephanyCS1** [comment]
***@***.*** este es mi correo :D gracias

El mar, 8 nov 2022 a la(s) 12:35, cmjosueh ***@***.***)
escribió:

> Tengo mas conocimientos en el backend, en bases de datos tengo experiencia
> en MySQL y PostgreSQL, el lenguaje de programación: Python y un año de
> experiencia con la plataforma de Azure, estuve en el curso AZ-500 y obtube
> los 80 puntos para la rifa de certificación. En los comentarios pon el
> nombre de tu usuario de git para que hagamos equipo
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/Launch-X-Latam/teams/mision-backend/discussions/27>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ3KUSYBSJUBAJ75Z4DSBPTWHKFPPANCNFSM6AAAAAAR2SISUI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
*Stephany F. Castro Salas*
Ingeniera Ambiental
Esp. Sistemas Integrados de Gestión QHSE
Esp. Gerencia De La Seguridad y Salud En El Trabajo
*3184002428*

*“Todos tus sueños pueden hacerse realidad si tienes el coraje de
perseguirlos”.Walt Disney*

Evita imprimir este mensaje si no es estrictamente necesario.

De esta manera ahorras agua, energía y recursos forestales

**Comment by AnaCatherineC** [comment]
Hola me gustaría unirme a tu grupo  , mi nombre de Github es AnaCatherineC ,me dices si formaré parte del grupo ,Saludos a todos n.n
